ch10.md

```diff
@@ -251,10 +244,10 @@ Like the other mathematical constructs we've explored, applicative functors hold
 To demonstrate:
 
 ```js
-  var tOfM = compose(Task.of, Maybe.of);
+const tOfM = compose(Task.of, Maybe.of);
 
-  liftA2(liftA2(_.concat), tOfM('Rainy Days and Mondays'), tOfM(' always get me down'));
-  // Task(Maybe('Rainy Days and Mondays always get me down'))
+liftA2(concat, tOfM('Rainy Days and Mondays'), tOfM(' always get me down'));
```

Comment:
I think we've lost one `lift` here ^.^. Since we have nested `Applicative`s here and `concat` is working on `String`s, the double `lift` is indeed required.